packages/openapi2-parser/test/fixtures/circular-example.json

```diff
@@ -405,28 +405,25 @@
                         "content": "company"
                       },
                       "value": {
-                        "element": "object",
```

Comment:
This and the other JSON fixtures are results of the behaviour change for when a schema object which contains properties was coered as an object.